ftdetect/ps1.vim

```diff
@@ -0,0 +1 @@
+autocmd BufNewFile,BufRead *.ps*1 setfiletype ps1
```

Comment:
I use https://github.com/PProvost/vim-ps1 so I used `setfiletype` here instead of `set filetype`. Vim doesn't detect `*.ps*1` files by default so this should be fine.